app/src/main/java/com/kylecorry/trail_sense/tools/qr/ui/ScanQRFragment.kt

```diff
@@ -215,8 +288,12 @@ class ScanQRFragment : BoundFragment<FragmentScanTextBinding>() {
             bitmap.recycle()
         }
 
-        if (message != null) {
-            onQRScanned(message!!)
+        message?.let{
```

Comment:
I think this will be clearer with an if/else block

Reply:
Ok, fixed